src/views/catalog/utils/quick-create-vm.ts

```diff
@@ -47,6 +59,8 @@ export const quickCreateVM: QuickCreateVMType = async ({
 
   const vm = getTemplateVirtualMachineObject(processedTemplate);
 
+  const hasSC = await hasStorageClass(vm);
```

Comment:
please change var and function name to something more clear, if feels to me like hasSC is very generic and this is not the case here